internal/examples/zpl/main.go

```
@@ -83,5 +83,5 @@ func main() {
 
 	m.SetBorder(false)
 
-	m.OutputFileAndClose("internal/examples/internal/pdfs/zpl.pdf")
```

Ah, so this was silently failing because the path didn't exist.

---

Make sense, I will change it.